src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -118,6 +118,10 @@ def is_safetensors_compatible(filenames, passed_components=None, folder_names=No
         components.setdefault(component, [])
         components[component].append(component_filename)
 
+    # If there are no component folders check the main directory for safetensors files
+    if not components:
+        return any(".safetensors" in filename for filename in filenames)
```

Comment:
Would also mention the description of this function because this is not accounted for there. 

https://github.com/huggingface/diffusers/blob/0d9d98fe5f828694cd0830a6ae2fc659211fd138/src/diffusers/pipelines/pipeline_loading_utils.py#L99